### PR TITLE
Parent Signup and Parent Login Implementation

### PR DESCRIPTION
**Pull request by mackorooji8**
This PR implements the signup and login functionality of the app. It also completes github issue #63. To test I recommend:
1. deleting the app (so all data is clearered out)
2. Run the app
3. Test it out by signing up for an account, attempting to login with the wrong PIN, logging in with the correct PIN, using the back buttons, etc.

Note: I noticed if you leave the fields on the signup or login page blank and hit the green button, some issues/errors can occur. Still need to handle that if we want it to be perfect.

### CODE REVIEW

File: app/src/main/java/com/GrowthPlus/ParentSignup.java [hunk 1]
```diff
@@ -2,13 +2,129 @@
...
+    }
+
+    @Override
+    public void onClick(View view) {
+
+        int id = view.getId();
+
+        //if they click the signup button
+        if(id == R.id.parentSignupBtn){
+            //track what the user entered
```
Comment (RuthShryock) [inline app/src/main/java/com/GrowthPlus/ParentSignup.java:69]: To address clicking the login/signup button with empty input, you can make a if statement 
`if (input.getText.toString() != "")` follow the logic you have written
`else` toast message saying empty input

File: app/src/main/java/com/GrowthPlus/ParentLogin.java [hunk 2]
```diff
@@ -2,13 +2,121 @@
...
+    }
+
+    @Override
+    public void onClick(View view) {
+        int id = view.getId();
+
+        //the user clicked the login button
+        if(id == R.id.parentLoginBtn){
+
+            loginPinInputInteger = Integer.parseInt(loginPinInput.getText().toString());
```
Comment (RuthShryock) [inline app/src/main/java/com/GrowthPlus/ParentLogin.java:64]: add the same logic that I mentioned in the pervious comment here

**Review by RuthShryock — CHANGES_REQUESTED** [review]
Everything looks good, I left two comments to address the empty input bug

**Review by RuthShryock — APPROVED** [review]
Nice work!

**Review by JulioC-19 — APPROVED** [review]
It's working as intended, good job.

**Pull request merged**